packages/ag-charts-enterprise/src/components/popover/popover.ts

```diff
@@ -36,6 +36,7 @@ export abstract class Popover<Options extends PopoverOptions = PopoverOptions>
     }
 
     public hide() {
+        if (this.element.children.length === 0) return;
```

Comment:
It's due to annotations eagerly calling `colorPicker.hide()` whenever an annotation is clicked.

It does this because when you mouse down it fires `drag-start` and goes into the dragging state, then when you mouse up it fires `drag-end` and goes back into the default idle state.

Whenever it enters the idle start, it resets the stage by hiding overlays like the color picker and clearing other things.

Possibly, it shouldn't call hide overlays so eagerly, but this was the simpler fix.

Reply:
Okay that makes sense. SGTM.
It might be wise add this explanation in a comment.